cvpy/image/Image.py

```diff
@@ -80,43 +80,6 @@ def __reverse(a, axis=0):
         idx[axis] = slice(None, None, -1)
         return a[idx]
 
-    def __get_num_channels_and_dtype(data_type: ImageDataType):
```

Comment:
Was this really deleted? I think this is the second commit, I just want to be sure I understand that this IS now deleted and  not needed.

Reply:
Yes it was! Total mistake on my part which is why I had to commit again. It's deleted now and not needed.